### PR TITLE
Fix species description in encyclopedia

### LINKED ISSUE #4587: No automatic GUI description of species traits after py conversion?
Bug Report
==========

During a game on build aa45874 I saw Acirema homeworld and when examining the Acirema species in pedia, I saw there is nothing about their traits, while I was sure there used to be plenty listed (like better industry, research, ship shields etc.).

Tried to see what has changed and I see Acirema have been moved to focs.py from focs.txt. I've noticed Abaddoni also have been moved, and sure enough, they lack those descriptions as well in GUI, including the pop up that you get when starting new game and selecting species you want to play. Starting the game with Abaddoni however shows bad research and great influence in action as expected, even if those are not listed. So seems to only be GUI issue.

Also Banforo and Ancient Guardians have this issue, also being python-ported

Expected Result
---------------

Species like Acirema or Abaddoni have their traits ++, -- listed in the GUI in their entry, or in case in Abaddoni, in the pop up when hovering over them in initial species selection drop down

Steps to reproduce
------------------

You don't even have to start game, you can launch Pedia from main menu, search for e.g. Banforo, and see no traits listed that show stuff like Good Industry. Additionally this also applies for Abaddoni and the drop down where you select starting species

### PR DESCRIPTION
**Pull request by o01eg**
Fixes #4587.

Also adds description to effect group dump.

### CODE REVIEW

File: parse/EffectPythonParser.cpp [hunk 1]
```diff
@@ -176,15 +176,19 @@ namespace {
         if (kw.has_key("accountinglabel")) {
             accountinglabel = py::extract<std::string>(kw["accountinglabel"])();
         }
-        // ToDo: implement other arguments later
+
+        std::string description;
+        if (kw.has_key("description")) {
+            description = py::extract<std::string>(kw["description"])();
+        }
```
Comment (geoffthemedio) [inline parse/EffectPythonParser.cpp]:
```suggestion
        if (kw.has_key("description"))
            description = py::extract<std::string>(kw["description"])();
```

Comment (o01eg) [reply parse/EffectPythonParser.cpp]: Fixed.

**Comment by geoffthemedio** [comment]
didn't test, but otherwise looks fine...

**Comment by Vezzra** [comment]
@o01eg, trying to cherry pick this gives me a merge conflict. Can you do the cherry picking, or provide a separate PR for the release branch?

**Comment by o01eg** [comment]
@Vezzra Opened #4598

**Pull request merged**